install/kubernetes/quick-install.yaml

```diff
@@ -180,7 +180,11 @@ data:
   operator-api-serve-addr: '127.0.0.1:9234'
 
   # Enable Hubble gRPC service.
-  enable-hubble: "false"
+  enable-hubble: "true"
+  # UNIX domain socket for Hubble server to listen to.
+  hubble-socket-path:  "/var/run/cilium/hubble.sock"
+  # An additional address for Hubble server to listen to (e.g. ":4244").
+  hubble-listen-address: ""
```

Comment:
Should this be here? Maybe https://github.com/cilium/cilium/pull/11880/files#diff-9d18c2e21e3f3468c2b60cdc13ecd7a2R457 should be set to `~` instead of `""`?